File: Code/Julia/Problem0005.jl
function euler5()
    N = 1
    while true
        divisors_count = 0
        for n in range(1, 20)
            if N % n == 0
                divisors_count += 1
            else
                N += 1
            end
        end
        if divisors_count == 20
            return N
        end
    end
end

print(euler5())

File: Code/Python/Problem0005.py
N = 1

while True:
    divisors_count  = 0
    for n in range(1,21):
        if N % n == 0:
            divisors_count += 1
        else:
            N += 1
            
    if divisors_count == 20:
        print(N)
        break    